src/mechanics/timeIntegration/NewmarkDirect.h

```diff
@@ -30,26 +30,11 @@ class NewmarkDirect : public NewmarkBase
     //! @brief constructor
     NewmarkDirect(StructureBase* rStructure);
 
-    void SetMinLineSearchStep(double rMinLineSearchStep)
```

Comment:
I didn't know such a method existed! It may not be used atm, but it's not useless though. After this change, I would have to recompile Newmark if I wanted to change that.